packages/ionic/src/commands/cordova/run.ts

```diff
@@ -334,14 +331,17 @@ Just like with ${input('ionic cordova build')}, you can pass additional options
       }
     }
 
+    const buildOpts: IShellRunOptions = { };
+    // ignore very verbose compiler output on stdout unless --verbose
+    buildOpts.stdio = options['verbose'] ? 'inherit' : ['pipe', 'ignore', 'pipe'];
```

Comment:
For builds we should show all output always.

Reply:
Ah, ok, I was duplicating what I saw in runServeDeploy for the handling of output - I'll remove all changes to runBuildDeploy as it was otherwise working wrt. #4165